pkg/spdk/engine.go

```
@@ -583,13 +664,42 @@
 	return res
 }
 
+func areNvmeSubsystemsEqual(old, new *api.NvmeSubsystem) bool {
+	if len(old.Paths) != len(old.Paths) {
```

golangci-lint (1.62.2)

SA4000: identical expressions on the left and right side of the '!=' operator

(staticcheck)